docs/cpp/c-cpp-properties-schema-reference.md

```diff
@@ -65,7 +65,9 @@ MetaDescription: Schema reference for C++ project settings in Visual Studio Code
   The full path to the compiler you use to build your project, e.g. `/usr/bin/gcc`, to enable more accurate IntelliSense. Arguments can be added to modify the includes/defines used, e.g. `-nostdinc++`, `-m32`, etc., but paths with spaces must be surrounded by double quotes `"` if arguments are used. The extension will query the compiler to determine the system include paths and default defines to use for IntelliSense.
 
 - `intelliSenseMode`
-  The IntelliSense mode used by the IntelliSense engine. `msvc-x64` maps to Visual Studio mode with 64-bit pointer sizes. `clang-x64` maps to CLang mode with 64-bit pointer sizes. `gcc-x64` maps to GCC mode with 64-bit pointer sizes. If not set or if the `${default}` mode is used, the extension will choose the default for that platform. Windows defaults to `msvc-x64`, Linux defaults to `gcc-x64`, and macOS defaults to `clang-x64`.
+  The IntelliSense mode to use that maps to an architecture-specific variant of MSVC, gcc, or Clang. 
```

Comment:
platform is one word.
Note that Markdown will treat this as one paragraph even through you have added carriage returns.
Consider using a bulleted list for the default list, it will be much easier for users to quickly parse the list for their platform

Reply:
Thank you, I made the changes